l10n_nl_xaf_auditfile_export/models/ir_qweb.py

```diff
@@ -13,7 +13,8 @@ class IrQwebAuditfileStringWidget999(models.AbstractModel):
     @api.model
     def value_to_html(self, value, options):
         value = value[: self._max_length] if value else ""
-        return super().value_to_html(value, options)
+        res = super().value_to_html(value, options)
+        return str(res)  # From markup to string
```

Comment:
I did some tests with the help of https://github.com/OCA/l10n-netherlands/pull/399, and indeed I'm running into the `xmlParseEntityRef: no name` error in cases where a field contains an ampersand.

However your fix only addresses cases where the widget is used. I found that changing `xml.unescape()` to `str(xml)` over [here](https://github.com/OCA/l10n-netherlands/blob/201d1ab5d46236ceb4bb5fb6a415bc9767f989b6/l10n_nl_xaf_auditfile_export/models/xaf_auditfile_export.py#L124) addresses the issue for all fields, regardless of the widget used (mine was [here](https://github.com/OCA/l10n-netherlands/blob/201d1ab5d46236ceb4bb5fb6a415bc9767f989b6/l10n_nl_xaf_auditfile_export/views/templates.xml#L253))

Reply:
I like your approach, I think is the correct way. Is it ok for you to fix it directly in your PR, so that we can close this one?

Reply:
Let's keep both things separated.

Reply:
Now done in https://github.com/OCA/l10n-netherlands/pull/410